project/src/components/place-card/place-card.tsx

```diff
@@ -4,18 +4,18 @@ import {AppRoute} from '../../const';
 
 type PlaceCardProps = {
   offers: OfferType;
-  onActiveOfferChange: (offer: OfferType | null) => void;
+  onPlaceCardHover: (offer: OfferType | null) => void;
 }
 
 function PlaceCard (props: PlaceCardProps): JSX.Element {
-  const {offers, onActiveOfferChange} = props;
+  const {offers, onPlaceCardHover} = props;
   const {previewImage, title, price, rating, type, id, isPremium} = offers;
 
   return (
     <article
       className="cities__place-card place-card"
-      onMouseOver = {() => onActiveOfferChange(offers)}
-      onMouseOut = {() => onActiveOfferChange(null)}
+      onMouseOver = {() => onPlaceCardHover(offers)}
```

Comment:
```suggestion
      onMouseOver = {() => onPlaceCardHover(offer)}
```
только тут в единственном числе. У тебя же офер один)